src/components/article/TranslatedArticlePage.tsx

```diff
@@ -180,9 +201,20 @@ export default function ({
                 {getDistanceLabel(modifiedAt, t.dates.updated)}
               </Badge>
             </Dates>
+            <BottomNavigation>
+              <Button
+                style={{ marginRight: "auto" }}
```

Comment:
Don't use inline styles. Add it to BottomNavigation styled. Mixing inline with not inline approach can create problem with maintenance later.